### DISCUSSION TITLE
[alpha] rxfire with Firebase Alpha SDK

### DISCUSSION THREAD

**Discussion by retronav** [firebase/firebase-js-sdk#4737] [General]
I was wondering if there were any plans for making rxfire usable with the new Firebase Alpha SDK.
Right now I'm implementing some functions to be used internally in my app, so maybe I could help on this.
BTW thanks a lot for the new light and modern SDK! Good job :)

**Comment by jamesdaniels** [comment]
Thanks for expressing interest, we have some ongoing work over at our new dedicated repository for RxFire https://github.com/FirebaseExtended/rxfire/pull/1, the most recent canary cut to that branch to NPM is `0.500.0-canary.3665c17`. At the very least feedback would be much appreciated.

**Reply by retronav** [reply]
Yes, I'll dabble with it today
